src/common/depthmap_toolkit/depthmap.py

```diff
@@ -9,7 +9,7 @@
 
 def export(type, filename):
     if type == 'obj':
-        utils.exportOBJ('export/' + filename)
+        utils.exportOBJ('export/' + filename, 1)
```

Comment:
To improve readability

```suggestion
        utils.exportOBJ('export/' + filename, triangulate=True)
```